sandboxed_containers/upgrade-sandboxed-containers.adoc

```diff
@@ -0,0 +1,25 @@
+[id="upgrade-sandboxed-containers"]
+= Upgrade {sandboxed-containers-first}
+include::modules/common-attributes.adoc[]
+:context: upgrade-sandboxed-containers
+
+toc::[]
+
+You can upgrade the components of {sandboxed-containers-first} by upgrading {sandboxed-containers-operator} and {sandboxed-containers-first} artifacts.
+
+[id="sandboxed-containers-upgrade-operator"]
+== Upgrade {sandboxed-containers-operator}
+
+You can use the OpenShift Lifecycle Manager (OLM) to manually or automatically upgrade the {sandboxed-containers-operator}. You can select manual or automatic upgrade during the initial deployment. In the context of manual upgrades, the web console shows the available updates which can be installed by the cluster administrator.
+
+.Additional resources
+* xref:../operators/admin/olm-upgrading-operators.adoc#olm-upgrading-operators[Upgrading installed Operators]
+
+[id="sandboxed-containers-upgrade-artifacts"]
+== Upgrade the {sandboxed-containers-first} artifacts
+
+The {sandboxed-containers-first} artifacts are deployed onto the cluster using {op-system-first} extensions.
+
+//Update the Red Hat CoreOS extension with the updated link once PR is approved for the Understanding section.
+
+The {op-system} extension `sandboxed containers` contains the required components to run Kata Containers such as the Kata containers runtime, the hypervisor QEMU, and its dependencies. The extension is upgraded when upgrading the cluster to a new release of {product-title}.
```

Comment:
I think "its dependencies" is unclear here. The way it reads, it could be referring to the extension, the runtime, or the QEMU. (I assume it's the dependencies of the extension but I'm not certain.) Can you you clarify somehow?